Library/Formula/coreutils.rb

```diff
@@ -29,6 +29,7 @@ class Coreutils < Formula
     depends_on "texinfo" => :build
     depends_on "xz" => :build
     depends_on "wget" => :build
+    depends_on "gperf" => :build
```

Comment:
Please include a log of the failed build without your patch using `brew gist-logs coreutils`.